apps/lcs/behaviours/custom-redirect.js

```diff
@@ -1,9 +1,16 @@
+const { shouldRedirectToBefore1988 } = require('../../../utils/index.js');
+
 module.exports = superclass => class extends superclass {
   successHandler(req, res, next) {
     const currentRoute = req.form.options.route;
     const action = req.params.action;
 
     if (req.sessionModel.get('isCurrentTenant') && currentRoute === '/tenant-address' && action === 'edit') {
```

Comment:
I would suggest moving `req.sessionModel.get('isCurrentTenant')` to the end to reduce the amount of checks 

`if (currentRoute === '/tenant-address' && action === 'edit' && req.sessionModel.get('isCurrentTenant') {`